codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/customization/AwsCustomGoDependency.java

```diff
@@ -29,8 +29,11 @@
             "service/apigateway/internal/customizations", "agcust");
     public static final GoDependency GLACIER_CUSTOMIZATION = aws(
             "service/glacier/internal/customizations", "glaciercust");
+
     public static final GoDependency S3_SHARED_CUSTOMIZATION = awsModuleDep(
             "service/internal/s3shared", null, "s3shared");
+    public static final GoDependency ACCEPT_ENCODING_CUSTOMIZATION = awsModuleDep(
+            "service/internal/accept-encoding", null, "acceptencodingcust");
```

Comment:
Won't this initially generate an invalid hash e.g. `v0.25.1-` ? since these are new modules. Maybe the `awsModuleDep` should use a different version, e,g just regular `gohash` instead of sharing the `AWS_SDK` value?

Reply:
Updated awsModuleDep to use a separate version from AWS_SDK.